stores/moderationSearchHealthcareProfessionalsStore.ts

```diff
@@ -0,0 +1,49 @@
+import { gql } from 'graphql-request'
+import { defineStore } from 'pinia'
+import { type Ref, ref } from 'vue'
+import { gqlClient } from '../utils/graphql.js'
+import type { HealthcareProfessional } from '~/typedefs/gqlTypes'
+
+export const useModerationSearchHealthcareProfessionalsStore = defineStore(
+    'moderationSearchHealthcareProfessionalsStore',
+    () => {
+        const healthcareProfessional: Ref<HealthcareProfessional[]> = ref([])
+        async function getHealthcareProfessionals(id: string) {
+            const healthcareProfessionalsResult = await queryHealthcareProfessionalById(id)
+            healthcareProfessional.value = healthcareProfessionalsResult
+        }
+
+        return {
+            getHealthcareProfessionals,
+            healthcareProfessional
+        }
+    }
+)
+
+async function queryHealthcareProfessionalById(id: string) {
+    try {
+        const healthcareProfessionalId = {
+            healthcareProfessionalId: id
+        }
+        const result = await gqlClient.request<{ healthcareProfessional: HealthcareProfessional[] }>(
+            getHealthcareProfessionalsGqlQuery,
+            healthcareProfessionalId
+        )
+        return result.healthcareProfessional ?? []
+    } catch (error: unknown) {
+        console.log(`Error querying the healthcare professionals: ${JSON.stringify(error)}`)
```

Comment:
```suggestion
        console.log(`Error querying healthcare professional ${id}: ${JSON.stringify(error)}`)
```
Eventually when we have a logging service it could be really helpful to include the `id` of the healthcare professional that caused the error.